binrz/man/rz-test.1

```diff
@@ -1,60 +1,64 @@
-.Dd Mar 31, 2020
+.Dd Jan 24, 2024
 .Dt RZ_TEST 1
 .Sh NAME
 .Nm rz-test
 .Nd rizin regression testsuite
```

Comment:
Not just `testsuite` but something like `testsuite running tool`. @wargio @kazarmy do you have better ideas for the description?